config/crd/bases/config.nri_topologyawarepolicies.yaml

```diff
@@ -94,6 +94,19 @@ spec:
                     - classes
                     type: object
                 type: object
+              defaultCPUPriority:
+                default: none
+                description: |-
+                  DefaultCPUPriority (high, normal, low, none)
+                  This parameter is passed to CPU allocator when allocating CPUs exclusively.
```

Comment:
Wait, so is it only for exclusive allocation? should it be then reflected in the name? From UX perspective, it should be a valid case for container request "low" priority cpus even from shared pool(s). e.g. E-cores on hybrids.

Reply:
That is a good question. This PR does not implement low-priority shared CPU allocation. 

Currently I don't know if I will be able to stretch the limits of the T-A policy enough to implement that without breaking stuff left and right. However, if it turns out to be possible with a reasonable effort, then I'd like to have a single configuration preference and a single annotation for controlling both exclusive and shared CPU allocation in this regard (both to keep configuration simpler and to avoid breaking configuration compatibility).

That is why I chose to reflect the semantics in the description of the configuration option and not in the name.

Reply:
then probably better to rephrase comment and docs for this option that it is **currently** affecting only exclusive allocation. might change in future.

Reply:
Updated.